File: Submissions/26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    
    public int removeDuplicates(int[] nums) {
        List<Integer> L = new ArrayList<>();
        char c;
        int k=-1;
        for(int i=0; i<nums.length; i++)
        {
            if(k == -1 || L.get(k) != nums[i])
                {L.add(nums[i]); k++;}   
        }

        for(int i=0; i<=k; i++)
        nums[i] = L.get(i);

        return k+1;
    }
}

File: Submissions/26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        L = []

        for item in nums:
            if L==[] or L[-1] != item:
                L.append(item)
        
        k = len(L)
        
        nums[:k] = L
        
        return k
        